appengine/flexible/HelloWorld/Properties/launchSettings.json

```diff
@@ -8,20 +8,21 @@
     }
   },
   "profiles": {
-    "IIS Express": {
-      "commandName": "IISExpress",
+    "HelloWorld": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,

```

Comment:
nit: The only real change here seems to be the setting of the `dotnetRunMessages` field. The rest is just because the order of the progiles changed. Would you mind reverting the order change so we see the real change?

Reply:
Done.